wp-content/themes/humanity-theme/patterns/article.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Title: Article Pattern
+ * Description: Article pattern for the theme
+ * Slug: amnesty/article
+ * Inserter: yes
+ */
+
+$sidebar_is_enabled = amnesty_get_meta_field( '_disable_sidebar' ) !== '1';
+$class_name         = $sidebar_is_enabled ? 'has-sidebar' : '';
+
+?><!-- wp:group {"tagName":"article","className":"article <?php print esc_attr( $class_name ); ?>"} -->
+<article class="wp-block-group article <?php print esc_attr( $class_name ); ?>">
+<!-- wp:pattern {"slug":"amnesty/hero"} /-->
```

Comment:
The hero should probably be in the main template below the header template part, as there's a lot of extra whitespace in the page as it is now:
![](http://bigbite.im/i/25tqWI+)